tests/Unit/Config/ConfigServiceProviderTest.php

```diff
@@ -19,6 +19,9 @@ class ConfigServiceProviderTest extends ServiceProviderTest
 {
     use ArraySubsetAsserts;
 
+    private array $configVarsWhereNullIsPruned = ['themes', 'tshirt_sizes', 'headers',
+                                                  'header_items', 'footer_items', 'locales'];
```

Comment:
We have still linter errors in this file
```
tests/Unit/Config/ConfigServiceProviderTest.php:
	23: Wrong amount of left-padding spaces(want multiple of 4)
	36: Wrong amount of left-padding spaces(want multiple of 4)
	37: Wrong amount of left-padding spaces(want multiple of 4)
	39: Wrong amount of left-padding spaces(want multiple of 4)
	40: Wrong amount of left-padding spaces(want multiple of 4)
	48: Wrong amount of left-padding spaces(want multiple of 4)
	51: Wrong amount of left-padding spaces(want multiple of 4)
	52: Wrong amount of left-padding spaces(want multiple of 4)
	53: Wrong amount of left-padding spaces(want multiple of 4)
	54: Wrong amount of left-padding spaces(want multiple of 4)
	66: Wrong amount of left-padding spaces(want multiple of 4)
```

Reply:
Weird, `docker exec engelsystem_dev-es_workspace-1 composer run phpcs` (as suggested in DEVELOPMENT.md) doesn't report these errors.
Will fix.